src/CabanaPD_Solver.hpp

```diff
@@ -537,22 +505,9 @@ class SolverFracture
     using NeighborView = typename Kokkos::View<int**, memory_space>;
     NeighborView mu;
 
-    using base_type::comm_time;
-    using base_type::energy_time;
-    using base_type::force_time;
-    using base_type::init_time;
-    using base_type::integrate_time;
-    using base_type::output_time;
-    using base_type::total_time;
-
-    using base_type::comm_timer;
-    using base_type::energy_timer;
-    using base_type::force_timer;
-    using base_type::init_timer;
-    using base_type::integrate_timer;
-    using base_type::output_timer;
-    using base_type::total_timer;
-
+    using base_type::_init_time;
+    using base_type::_init_timer;
+    using base_type::_step_timer;
```

Comment:
Why do we have  "_init_time" in addition "_init_timer" to but not "_step_time"

Reply:
This gets used in a few different places unlike other timers - see the prenotch section in this file